requirements.txt

```diff
@@ -1,4 +1,5 @@
 aniso8601==9.0.1
+antlr4-python3-runtime==4.7.2
```

Comment:
Mean
Was this version of `antlr` throwing warning w/ the version of SymPy we have?

Reply:
nah that was `4.9.2`